changelog/range-error.md

```diff
@@ -0,0 +1,54 @@
+Out of bounds array access now gives a fluent error message
```

Comment:
POV: "fluent" is a bit vague, I would advise to either phrase it as "now includes indices" or mention "better" or "improved" error message. But that's POV, I think the message is good enough as-is.

Reply:
I borrowed the term 'fluent' from the common 'fluent asserts' you find in unit testing packages, but I agree it's not the best word. I'm thinking changing it to "informative" or, indeed, "better".